lib/pages/login_page.dart

```diff
@@ -0,0 +1,90 @@
+import 'dart:ui';
+
+import 'package:email_validator/email_validator.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+class LoginPage extends StatefulWidget {
+  LoginPage({Key key}) : super(key: key);
+
+  @override
+  _LoginPageState createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: Center(
+        child: Container(
+            constraints: BoxConstraints.expand(),
+            decoration: BoxDecoration(
+              image: DecorationImage(
+                  fit: BoxFit.cover,
+                  image: AssetImage('assets/images/sign_in_background.png')),
+            ),
+            child: ClipRRect(
+              child: BackdropFilter(
+                  filter: ImageFilter.blur(sigmaX: 12, sigmaY: 12),
+                  child: Padding(
+                    padding: EdgeInsets.only(left: 24, right: 24),
+                    child: Column(
+                      mainAxisAlignment: MainAxisAlignment.start,
+                      crossAxisAlignment: CrossAxisAlignment.stretch,
+                      children: [
+                        Container(
+                          margin: const EdgeInsets.only(top: 150.0),
+                          child: SvgPicture.asset('assets/images/logo.svg'),
+                        ),
+                        SizedBox(height: 100),
+                        TextFormField(
+                          keyboardType: TextInputType.emailAddress,
+                          decoration: InputDecoration(
+                              labelStyle: TextStyle(color: Colors.white30),
+                              floatingLabelBehavior:
+                                  FloatingLabelBehavior.never,
+                              border: OutlineInputBorder(
+                                  borderSide: BorderSide.none,
+                                  borderRadius: BorderRadius.circular(12.0)),
+                              fillColor: Colors.white24,
+                              filled: true,
+                              labelText: 'Email'),
+                          style: TextStyle(color: Colors.white),
+                          validator: (value) {
+                            if (value == null || value.isEmpty) {
+                              return "Please enter your email";
+                            } else if (!EmailValidator.validate(value)) {
+                              return "Please enter a valid email";
+                            }
+                            return null;
```

Comment:
It's better for reading, IMO
```suggestion
                            if (value == null || value.isEmpty) {
                              return "Please enter your email";
                            } 
                            
                            if (!EmailValidator.validate(value)) {
                              return "Please enter a valid email";
                            }
                            return null;
```

Reply:
there is no convention on this yet but I guess I can adapt it for now.